tests/core/operation/test_publish_operation.py

```diff
@@ -102,6 +102,15 @@ def test(self):
                 },
             )
 
+            qs_stub.add_response(
```

Comment:
@tdilauro :  to answer your question:  in the test I have added the delete call here.   Previously there wasn't a test for "update", just the initial create.  Now there is just one path  (delete and create) for dashboards.   Does that answer your question?